### PR TITLE
Custom authoring tweaks

### PR DESCRIPTION
**Pull request by sfentress**
A collections of small changes to the authoring pages based on QA and feedback.

### CODE REVIEW

File: src/components/authoring-mice.tsx [hunk 1]
```diff
@@ -278,7 +279,8 @@ export const MiceAuthoringComponent: React.FC<IProps> = ({initialAuthoring}) =>
                       Breed with mutations
                     </label>
                     <div className="instruction nested-instruction">
-                      <b>Selected:</b> Mutations may randomly occur in offspring, and are passed down.<br/>
+                      <b>Selected:</b> Mutations in fur color genes may randomly occur in offspring, and are
+                      passed down.<br/>
```
Comment (mklewandowski) [inline src/components/authoring-mice.tsx:283]: Don't believe the comma is needed.

File: src/components/authoring-mice.tsx [hunk 2]
```diff
@@ -401,7 +403,8 @@ export const MiceAuthoringComponent: React.FC<IProps> = ({initialAuthoring}) =>
...
```
Comment (mklewandowski) [inline src/components/authoring-mice.tsx:407]: Same here in regards to the comma

File: src/components/authoring-peas.tsx [hunk 3]
```diff
@@ -78,7 +78,8 @@ export const PeasAuthoringComponent: React.FC<IProps> = ({initialAuthoring}) =>
                     Breed with mutations
                   </label>
                   <div className="instruction nested-instruction">
-                    <b>Selected:</b> Mutations may randomly occur in offspring, and are passed down.<br/>
+                    <b>Selected:</b> Mutations in pea shape genes may randomly occur in offspring, and are
+                    passed down.<br/>
```
Comment (mklewandowski) [inline src/components/authoring-peas.tsx:82]: Same here in regards to the comma

**Review by mklewandowski — APPROVED** [review]
👍 LGTM!  I made a minor comment about comma usage - you can make the call on that.  Code looks good.

File: src/components/authoring-mice.tsx (hunk 1 above)
Comment (sfentress) [reply src/components/authoring-mice.tsx:283]: You're right.

**Pull request merged**